reactjs-todo-dv/client/components/davinci-client/flow-button.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * forgerock-sample-web-react
+ *
+ * flow-button.js
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+import React from 'react';
+
+export default function FlowButton({ collector, flow, renderForm }) {
+
+  const clickHandler = async () => {
+    const node = await flow(collector.output.key);
+    renderForm(node);
+  };
+
+  return (
```

Comment:
May I ask why the spinner component was removed?

Reply:
i was thinking the flow buttons might actually be links at some point but it can be added back.  im still trying to get my head around how the sdk behaves vs what the flow is built like in DaVinci.  This OOTB flow uses buttons.

Reply:
I'm not super educated on DaVinci flows, but with my understanding, I wouldn't think these "flow buttons" would ever be links as they don't result in a navigation (a change in URL), but an internal change of reference of the form to render.

Were you thinking that click of a flow button would result in a navigation change to a new page/view for rendering the new flow? Or, that the internal form itself just renders different form elements?

Reply:
I was looking at the example app in javascript sdk repo and it was using links rather than buttons, that was the only reason.

Reply:
Oh, sorry for the confusion. I may have called them links, but technically, they are buttons. The are links in just the visual sense: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/e2e/davinci-app/components/flow-link.ts#L12.